packages/fmg-nitro-adjudicator/contracts/NitroAdjudicator.sol

```diff
@@ -139,21 +195,40 @@ contract NitroAdjudicator {
             "Transfer: outcome must be present"
         );
 
-        uint channelAffordsForDestination = affords(destination, outcomes[channel], holdings[channel]);
+        uint channelAffordsForDestination = Library.affords(destination, outcomes[channel], holdings[channel][token]);
```

Comment:
It doesn't look like `affords` is taking the token into account here. It'll need to do that for this to work correctly.

Reply:
Addressed in 0da44c1